### DISCUSSION TITLE
Dedicated NDI Filters with 4.12.0?

### DISCUSSION THREAD

**Discussion by jbcurler2010** [DistroAV/DistroAV#927] [General]
Haven't done a full test but has anyone seen issues with Dedicated Filters with the new version? Only had time to set one up today. The NDI Stream shows on the network but no image. I'll do more tomorrow but wanted to put it out there.

**Comment by Trouffman** [comment]
@jbcurler2010 can you check of this it still a problem on the latest version ?
If not : close this discussion :D

**Reply by jbcurler2010** [reply]
I'll give it a look today. Also, just curious who is responsible for updating the OBS Plugins page with the info for 4.13.2?

**Reply by Trouffman** [reply]
Which page are you referring to ? The GitHub page should be good.
Or the one on the OBS project forum ?

**Reply by jbcurler2010** [reply]
Yes. Here...

https://obsproject.com/forum/resources/obs-ndi-newtek-ndi%E2%84%A2-integration-into-obs-studio.528/updates

**Reply by jbcurler2010** [reply]
Also, confirmed that the original issue has been resolved here. Tested with OBS 30.1.2, NDI 6.0.0.0, and OBS NDI 4.13.2.
